cfpb/consumerfinance.gov
# Paying for College: Remove JS file from static disclosure technote page

The Paying for College Disclosures technote page was pulling in the app's JS file. This is a static text page, so the script had the error:

<img width="524" alt="Screen Shot 2022-12-22 at 8 47 34 AM" src="https://user-images.githubusercontent.com/704760/209147864-72b1e985-7be9-4d14-bc76-87912e0f9e7f.png">

## Removals

- Remove JS script from Paying for College Disclosures technote page.

## How to test this PR

1. Visit http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/ and see there's no error.
4. Navigate to [Dynamic Disclosures locally](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAHCA&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=0&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=4D0B9265B45716DF4B81EEB4BE117E1DF69871AF) and verify that the technote link ("Get details about how this tool works") link takes you to the tech note page and there's no error.
